plugins/ci/pkg/models/scan-errors.go

```diff
@@ -0,0 +1,143 @@
+package models
+
+import (
+	"github.com/hashicorp/go-multierror"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	ScanErrorsReportVersion             = "0.0.1"
+	ScanErrorsReportDefaultKind         = "CIErrorWithoutContext"
+	ScanErrorsReportDefaultResourceName = "unknown"
+	ScanErrorsReportDefaultErrorContext = "performing an action in CI"
+)
+
+// ScanErrorResult contains a single error encountered during a scan.
+// This satisfies the GO Error interface, and provides additional error context
+// to be included in scan-errors report action items.
+type ScanErrorsReportResult struct {
+	// IF adding a field to this struct, also update the FillUnsetFields
+	// receiver below!
+	Kind         string  `json:"kind"`
+	ResourceName string  `json:"resourceName"`
+	ErrorMessage string  `json:"errorMessage"` // error message returned during a scan
+	ErrorContext string  `json:"errorContext"` // where were we / what was happening when the error occurred
+	Filename     string  // filename being scanned that relates to this error
```

Comment:
Looks like it is missing the `json` tag for `Filename`

Reply:
Ah, it seemed to use the member name by default, but I've added the tag for clarity. Thanks.

Reply:
ohh, I thought that it would use pascal case by default if not specified... that is why I mentioned 😅